backend/controller/timeline/events_call.go

```diff
@@ -92,45 +146,42 @@ func (s *Service) RecordCall(ctx context.Context, call *Call) {
 		Response:         responseBody,
 		Error:            errorStr,
 		Stack:            stack,
-	})
-	if err != nil {
-		logger.Errorf(err, "failed to record call")
 	}
 }
 
-func (s *Service) insertCallEvent(ctx context.Context, call *CallEvent) error {
-	var sourceModule, sourceVerb optional.Option[string]
-	if sr, ok := call.SourceVerb.Get(); ok {
-		sourceModule, sourceVerb = optional.Some(sr.Module), optional.Some(sr.Name)
-	}
-	var requestKey optional.Option[string]
-	if rn, ok := call.RequestKey.Get(); ok {
-		requestKey = optional.Some(rn.String())
+func callEventToCall(event *CallEvent) *Call {
+	var response either.Either[*ftlv1.CallResponse, error]
+	if eventErr, ok := event.Error.Get(); ok {
+		err := errors.New(eventErr)
```

Comment:
Since this `err` is only used on the line below, what do you think about inlining it as `response = either.RightOf[*ftlv1.CallResponse](errors.New(eventErr))`?